src/components/Alteration/AlterationSummary.vue

```diff
@@ -72,7 +72,7 @@
     <!-- Name Translation -->
     <template v-if="haveNameTranslationsChanged">
       <v-divider class="mx-4" />
-      <div class="section-container name-translation-summary">
+      <div class="name-translation-summary">
```

Comment:
The NameTranslation and OfficeAddress components have a section-container class already -- needed so we can just drop them in the YourCompany wrapper. So that class has to be removed here.

I checked the other components that we can both drop in the wrapper AND is on the Summary pages and these are the only 2 (but they are on several summary pages).